### PR TITLE
updating esc readme

### PR DESCRIPTION
**Pull request by interurban**
Adding metadata and updating features for recent esc launches. Adding links to resources, getting started and docs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,61 +1,120 @@
...
+
+Pulumi ESC enables teams to aggregate secrets and configuration from many sources, manage hierarchical collections of configuration and secrets as environments, and consume those configuration and secrets from a variety of different infrastructure and application services.
 
-Pulumi ESC enables teams to aggregate secrets and configuration from many sources, manage hierarchical collections of configuration and secrets ("environments"), and consume those configuration and secrets from a variety of different infrastructure and application services.  Pulumi ESC works hand-in-hand with Pulumi IaC to simplify configuration management, but also works independently from Pulumi IaC, as a solution for managing environments, secrets and configuration for any application or infrastructure project.
+Pulumi ESC works hand-in-hand with Pulumi IaC to simplify configuration management, but also works independently from Pulumi IaC, as a solution for managing environments, secrets and configuration for any application or infrastructure project.
```
Comment (arunkumar611) [inline README.md:16]: would reword this to say something like "It simplifies secrets and configurations for all your application and infrastructure, including Pulumi IaC"

File: README.md [hunk 2]
```diff
@@ -1,61 +1,120 @@
...
 
 For example, the Pulumi ESC CLI (`esc`) makes it possible to give your developers immediate, just-in-time authenticated and short-lived access to cloud credentials across any cloud provider with just a single command: `esc run aws-staging -- aws s3 ls`.
 
-![Pulumi ESC Overview GIF](./assets/esc.gif)
+![Pulumi's open source secrets management solution overview](./assets/esc_product_demo.gif)
```
Comment (arunkumar611) [inline README.md:20]: It would be great to update a new video - that first starts off with the UI and continues on to CLI, and then to code.

File: README.md [hunk 3]
```diff
@@ -1,61 +1,120 @@
...
 
-Pulumi ESC enables teams to aggregate secrets and configuration from many sources, manage hierarchical collections of configuration and secrets ("environments"), and consume those configuration and secrets from a variety of different infrastructure and application services.  Pulumi ESC works hand-in-hand with Pulumi IaC to simplify configuration management, but also works independently from Pulumi IaC, as a solution for managing environments, secrets and configuration for any application or infrastructure project.
+Pulumi ESC works hand-in-hand with Pulumi IaC to simplify configuration management, but also works independently from Pulumi IaC, as a solution for managing environments, secrets and configuration for any application or infrastructure project.
 
 For example, the Pulumi ESC CLI (`esc`) makes it possible to give your developers immediate, just-in-time authenticated and short-lived access to cloud credentials across any cloud provider with just a single command: `esc run aws-staging -- aws s3 ls`.
```
Comment (arunkumar611) [inline README.md:18]: The transition here from the previous paragraph doesn't feel smooth.

File: README.md [hunk 4]
```diff
@@ -1,61 +1,120 @@
...
 
-![Pulumi ESC Overview GIF](./assets/esc.gif)
+![Pulumi's open source secrets management solution overview](./assets/esc_product_demo.gif)
 
-Pulumi ESC is offered as a managed service as part of Pulumi Cloud, and this repo contains the implementation of the following key components of the ESC offering:
+Pulumi ESC is offered as a managed service as part of [Pulumi Cloud,](https://www.pulumi.com/product/pulumi-cloud/?utm_campaign=pulumi-esc-github-repo&utm_source=github.com) and this repo contains the implementation of the following key components of the ESC open source secrets and configuration management solution:
```
Comment (arunkumar611) [inline README.md:22]: Pulumi ESC is "also" offered?

File: README.md [hunk 5]
```diff
@@ -1,61 +1,120 @@
...
-* __Authentication and RBAC__:  Pulumi ESC brokers access to secrets and configuration that live in other systems, and so authentication and granular RBAC are critical to ensure robust access controls across your organization.  Pulumi ESC leverages the same Pulumi Cloud identity, RBAC, Teams, SAML/SCIM and scoped access tokens that are used for Pulumi IaC today, extending these all to managing access to Environments as well as Stacks.
-* __Configuration as Code__:  Environments are defined as YAML documents which can describe how to project and compose secrets and configuration, integrate dynamic configuration providers, and compute new configuration from other values (construing a URL from a DNS name, or concatenating multiple configuration values into a derived value).  The incredible flexibility of a code-based approach over traditional point-and-click interfaces allows Pulumi ESC to offer rich expressiveness for managing complex configuration.
+Pulumi ESC was born to address these problems and needs head on with the following features.
+
+### Pulumi ESC Features
```
Comment (arunkumar611) [inline README.md:87]: Feature right now seem like a mix of benefit vs. strictly feature. I believe this is ported over from the compete docs?

File: README.md (hunk 2 above)
Comment (interurban) [reply README.md:20]: Agree; I'll file a separate issue to track this.

File: README.md (hunk 4 above)
Comment (interurban) [reply README.md:22]: thx!

File: README.md (hunk 5 above)
Comment (interurban) [reply README.md:87]: yes, also luke's blogs etc.

File: README.md (hunk 3 above)
Comment (interurban) [reply README.md:18]: Suggestions?

File: README.md (hunk 1 above)
Comment (interurban) [reply README.md:16]: done

File: README.md (hunk 3 above)
Comment (arunkumar611) [reply README.md:19]: (Needs proper formatting, grammar check and links but here is my suggestion)

[Pulumi ESC](https://www.pulumi.com/product/esc/) is an open-source secrets management platform that tames secrets and configuration complexity across all your applications and infrastructure. 

With Pulumi ESC, teams can aggregate secrets and configurations from many sources, manage hierarchical collections of configurations and secrets as `environments`, and consume them through a variety of means including CLI, SDK, REST API, Pulumi Cloud Web Console and Pulumi-service provider <add links to all of them>. 

Pulumi ESC not only works great for your applications and IaC, including Pulumi IaC, it also makes your day-to-day developer workflow much more secure and streamlined. For example, you Pulumi ESC CLI (esc) makes it possible to give your developers immediate, just-in-time authenticated and short-lived access to cloud credentials across any cloud provider with just a single command.

 $ esc run aws-staging -- aws s3 ls.
 
 In this command, an ESC environment named aws-staging has all the necessary staging environment configuration and OIDC setup to connect to AWS. Running this command opens up a temporary environment and executes the `aws s3 ls` command in that temporary environment. The temporary AWS credentials are not stored anywhere, making it secure and also allowing you to dynamically switch between different environments.

**Review by arunkumar611 — APPROVED** [review]
Pre-approving.

File: README.md (hunk 3 above)
Comment (interurban) [reply README.md]: thx
